apps/api-extractor/src/collector/Collector.ts

```diff
@@ -511,41 +511,47 @@ export class Collector {
       this._calculateMetadataForDeclaration(astDeclaration);
     }
 
-    // We know we solved parentAstSymbol.metadata above
-    const parentSymbolMetadata: SymbolMetadata | undefined = astSymbol.parentAstSymbol
-      ? astSymbol.parentAstSymbol.metadata as SymbolMetadata : undefined;
-
-    const symbolMetadata: SymbolMetadata = new SymbolMetadata();
-
     // Do any of the declarations have a release tag?
     let effectiveReleaseTag: ReleaseTag = ReleaseTag.None;
 
     for (const astDeclaration of astSymbol.astDeclarations) {
       // We know we solved this above
       const declarationMetadata: DeclarationMetadata = astDeclaration.metadata as DeclarationMetadata;
-
       const declaredReleaseTag: ReleaseTag = declarationMetadata.declaredReleaseTag;
 
       if (declaredReleaseTag !== ReleaseTag.None) {
         if (effectiveReleaseTag !== ReleaseTag.None && effectiveReleaseTag !== declaredReleaseTag) {
           if (!astSymbol.isExternal) { // for now, don't report errors for external code
```

Comment:
Do we want new logic to apply to externals too?

Reply:
In the future, yes.  I'd consider it out of scope for your PR unless it's impacting you somehow.

The problem is that, if an external dependency has a (non-critical) mistake, that shouldn't prevent you from successfully building your own project.  Maybe you don't have any control over the upstream package.  Thus, if we report an error for external issues, we need to provide a way to suppress the errors in case they aren't solvable.  AE7 introduced limited support for suppressions via **api-extractor.json**, but it suppresses /all/ instances.  Ideally we'd want a little more granularity than that.  It's a good feature, but relatively low priority.